content/_data/changelogs/lts.yml

```diff
@@ -3913,6 +3913,590 @@
+    message: |-
+      Show in plugin manager when newer releases of plugins exist but aren't being offered due to unsatisfied requirements.
+  - type: rfe
+    category: rfe
+    pull: 4073
+    issue: 59136
+    authors:
+    - jetersen
+    - daniel-beck
+    message: |-
+      Show warnings for <b>deprecated</b> plugins in the update manager and administrative monitors.
+
+  # More user experience improvements
+  - type: rfe
+    category: rfe
+    references:
+      - issue: 62698
+      - pull: 4808
+      - pull: 4782
+      - issue: 4767
+      - issue: 62175
+    authors:
+    - fqueiruga
+    message: |-
+      Restyle hyperlinks and side panel widgets to have a more modern look & feel.
+      Update the styles for the links on the management page.
+  - type: rfe
+    category: rfe
+    pull: 4700
+    issue: 61973
+    authors:
+    - fqueiruga
+    message: |-
+      Revamp the sidebar task list with improved aesthetics and accessibility.
+  - type: rfe
+    category: rfe
+    pull: 4740
+    issue: 62421
+    authors:
+    - romenrg
+    message: |-
+      Improve accessibility by adding aria-labels for username and password fields in login forms.
+  - type: rfe
+    category: rfe
+    issue: 62750
+    pull: 4816
+    authors:
+    - fqueiruga
+    message: |-
+      Restyle and improve accessibility for the RSS feed bar.
+  - type: rfe
+    category: rfe
+    pull: 4757
+    issue: 62496
+    authors:
+    - EstherAF
+    - timja
+    message: |-
+      Convert arrow icons from images to CSS.
+  - type: rfe
+    category: rfe
+    pull: 4852
+    authors:
+    - oleg-nenashev
+    message: |-
+      Reorder configuration and delete buttons in the left sidepanel.
+  - type: rfe
+    category: rfe
+    pull: 4820
+    authors:
+    - timja
+    message: |-
+      Update Jenkins form styling in preparation for changing form layout.
+    references:
+      - issue: 56109
+      - url: https://issues.jenkins-ci.org/browse/JENKINS-62437
+        title: "Configuration UI Accessibility: Tables to Divs migration"
+  - type: rfe
+    category: rfe
+    pull: 4835
+    issue: 63002
+    authors:
+    - fqueiruga
+    message: |-
+      Revamp the tabs and tables with better spacing and a consistent color palette.
+  - type: rfe
+    category: rfe
+    pull: 4748
+    issue: 61806
+    authors:
+    - 95jonpet
+    - jglick
+    message: |-
+      Remove page generation timestamp from the footer.
+  - type: rfe
+    category: rfe
+    pull: 4840
+    issue: 25259
+    authors:
+    - tizki
+    message: |-
+      Add sorting to Old Data management tables.
+  - type: rfe
+    category: rfe
+    pull: 4466
+    issue: 20052
+    authors:
+    - res0nance
+    message: |-
+      Performance: Reduce number of calls and iterations through the list of jobs when displaying List Views.
+  - type: bug
+    category: bug
+    pull: 4771
+    authors:
+    - fqueiruga
+    message: |-
+      Restore background colors on new item and job config pages.
```

Comment:
This fixes a regression introduced in 2.239, so has no reason to be mentioned here.